app/src/main/java/net/gsantner/markor/activity/ActionOrderActivity.java

```diff
@@ -0,0 +1,226 @@
+        cu.tintMenuItems(menu, true, Color.WHITE);
+
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        int id = item.getItemId();
+        switch (id) {
+
+            case android.R.id.home:
+                // Return to settings
+                finish();
+                return true;
+
+            case R.id.action_reorder_accept:
+                ArrayList<String> reorderedKeys = new ArrayList<>();
+                for (int i : _adapter.order) reorderedKeys.add(_keys.get(i));
+                _textActions.saveActionOrder(reorderedKeys);
+                finish();
+                return true;
+
+            default:
+                return super.onOptionsItemSelected(item);
+        }
+    }
+
+    private void extractActionData() {
+
+        // Extract actions
+        int documentType = getIntent().getExtras().getInt("key");
+
+        switch (documentType) {
+            default:
+            case R.id.action_format_markdown:
+                _textActions = new MarkdownTextActions(this, null);
+                break;
+            case R.id.action_format_todotxt:
+                _textActions = new TodoTxtTextActions(this, null);
+                break;
+            case R.id.action_format_plaintext:
+                _textActions = new PlaintextTextActions(this, null);
+                break;
+        }
+
+        Map<String, ActionItem> actionMap = _textActions.getActiveActionMap();
+        _keys = new ArrayList<>(_textActions.getActionOrder());
+        _actions = new ArrayList<ActionItem>();
+
+        for (String key: _keys) {
+            _actions.add(actionMap.get(key));
+        }
+    }
+
+    class Adapter extends RecyclerView.Adapter<Holder> {
+        private List<ActionItem> _actions;
+        ArrayList<Integer> order;
+
+        Adapter(List<ActionItem> actions) {
+            super();
+            _actions = actions;
+
+            order = new ArrayList<>();
+            for (int i = 0; i <_actions.size(); i++) order.add(i);
+        }
+
+        @Override
+        public Holder onCreateViewHolder(ViewGroup parent, int viewType) {
+            return new Holder(getLayoutInflater().inflate(R.layout.action_reorder_item, parent, false));
+        }
+
+        @Override
+        public void onBindViewHolder(Holder holder, int position) {
+            holder.bindModel(_actions.get(order.get(position)));
+        }
+
+        @Override
+        public int getItemCount() {
+            return _actions.size();
+        }
+    }
+
+    static class Holder extends RecyclerView.ViewHolder {
+
+        private LinearLayout _row;
+
+        Holder(View row) {
+            super(row);
+            _row = (LinearLayout) row;
+        }
+
+        void bindModel(ActionItem action) {
+            ((ImageView) _row.getChildAt(0)).setImageResource(action.iconId);
+            ((TextView) _row.getChildAt(1)).setText(action.stringId);
+        }
+
+        void setHighlight() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5344500)